bndtools.builder/src/org/bndtools/builder/BndtoolsBuilder.java

```diff
@@ -113,15 +112,21 @@
 					.getDynamicReferences();
 			}
 
-			if (model == null) {
-				try {
-					model = Central.getProject(myProject);
-				} catch (Exception e) {
-					markers.deleteMarkers("*");
-				}
-				if (model == null)
-					return noreport();
+			Project ourModel = null;
+			try {
+				ourModel = Central.getProject(myProject);
+			} catch (Exception e) {
+				markers.deleteMarkers("*");
 			}
+			if (ourModel == null) {
+				markers.deleteMarkers("*");
```

Comment:
You now call `markers.deleteMarkers("*")` twice if you get an exception on `Central.getProject(myProject)`.

Reply:
Fixed.